lxd/storage/drivers/driver_lvm_volumes.go

```diff
@@ -244,6 +244,10 @@ func (d *lvm) HasVolume(vol Volume) bool {
 
 // FillVolumeConfig populate volume with default config.
 func (d *lvm) FillVolumeConfig(vol Volume) error {
+	// Copy volume.* keys from pool.
+	// Omit 'block.filesystem' and 'block.mount_options' as this one are dependant from volume.
+	d.fillVolumeConfig(vol, []string{"block.filesystem", "block.mount_options"})
```

Comment:
I don't understand the need for the 2nd argument on fillVolumeConfig, nor the comment here. `block.filesystem` would seem to be a setting that certainly should be copied from the pool volume settings if not explicitly set on the volume already.

Reply:
`fillVolumeConfig` copy all keys which was defined by `volumeRules` as this is generic method but values for this two keys are dependant from volume instance (check couple lines below) so I omit this two keys. But rigth now I'm thinking that I overcoplicated this a little bit. I think that we can remove the second parameter from `fillVolumeConfig` method and also remove this two keys from `volumeRules` and treat this as special case. Let me refactor this.

Reply:
We should make the 2nd argument a variadic form too.